acerola_shader_compiler.gd

```diff
@@ -9,7 +9,7 @@ var compute_shader_file_paths : Array = Array()
 var rd : RenderingDevice
 
 var shader_compilations = {}
-var shader_code_cache = {}
+var shader_compilation_time = {}
```

Comment:
I think this might be better named `shader_modified_time` or `raw_shader_modified_time`. It's possible the shader may be modified while this is not running, making modification time != compilation time.